src/Calculator/App.xaml

```diff
@@ -2179,7 +2179,27 @@
                                                                   AcceptsReturn="false"
                                                                   InputScope="Text"
                                                                   MaxLength="2048"
-                                                                  TextWrapping="NoWrap"/>
+                                                                  TextWrapping="NoWrap">
+                                            <Controls:MathRichEditBox.ContextFlyout>
```

Comment:
good thought.  Will add those.

Reply:
Turns out we can intercept the original context menu opening event and then add in our own to the returned result, but I'm seeing some crashes when I try this in C++ (though it appears to work in C# prototypes).  I think I want to do this as a followup though as it might take a bit of engagement with the xaml team to figure out why the C++ code doesn't work.

Reply:
Sounds good